vector/src/main/java/im/vector/activity/VectorMediasPickerActivity.java

```diff
@@ -1018,11 +1029,12 @@ public void onPictureTaken(byte[] data, Camera camera) {
                     // replace the high res picture with the preview one
                     // because the aspect ratio is not the same
                     // and the user would not understand that the mShotPicturePath image is not the previewed one
+                    /*
                     if (mIsAvatarMode && (null != mCameraTextureView.getBitmap())) {
                         ByteArrayOutputStream bos = new ByteArrayOutputStream();
                         mCameraTextureView.getBitmap().compress(Bitmap.CompressFormat.JPEG, AVATAR_COMPRESSION_LEVEL, bos);
                         data = bos.toByteArray();
-                    }
+                    }*/
```

Comment:
With Git we have full history, so it's OK to delete code instead of commenting it out if it's not useful :)